query-engine/js-connectors/playground/src/engines/types/Library.ts

```diff
@@ -29,27 +29,29 @@ export interface Query {
 
 // Same order as in rust js-connectors' `ColumnType`
 export const enum ColumnType {
-  Int32 = 0,
-  Int64 = 1,
-  Float = 2,
-  Double = 3,
-  Text = 4,
-  Enum = 5,
-  Bytes = 6,
-  Boolean = 7,
-  Char = 8,
-  Array = 9,
-  Numeric = 10,
-  Json = 11,
-  DateTime = 12,
-  Date = 13,
-  Time = 14
+  Int32,
+  Int64,
+  Float,
+  Double,
+  Numeric,
+  Boolean,
+  Char,
+  Text,
+  Date,
+  Time,
+  DateTime,
+  Json,
+  Enum,
+  Bytes,
+  // Set,
+  // Array,
+  // ...
 }
 
-export type Queryable = {
+export type Driver = {
```

Comment:
Same suggestion about using the term Connector rather than driver. I know we are abusing a bit the term connector across the two code bases. But I think it's more appropriate than driver in the end.